benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1074,15 +1074,13 @@ void GraphicsBenchmarkApp::Render()
     // Reset query
     frame.timestampQuery->Reset(/* firstQuery= */ 0, frame.timestampQuery->GetCount());
 
-    // Update scene data
-    frame.sceneData.viewProjectionMatrix = mCamera.GetViewProjectionMatrix();
 #if defined(PPX_BUILD_XR)
```

Comment:
probably better as a separate function in case other part of the code need to access the camera